### ISSUE TITLE
add MBO to the templates

### ISSUE THREAD

**Issue by kmexter** [marco-bolo/metadata-catalogue#31]
Marco Bolo should be added to all the templates as being the project under which the data were aggregated.
@dblear can recommend the value to add (the URL that points to MBO, or whether we just use the string MARCO-BOLO), but meanwhile @laurianvm can you decide what the term should be?
Something like partOf, usedIn

**Comment by pbuttigieg** [comment]
> This applies only to datasets created under MBO.

No, it also applies to any datasets aggregated, curated, modified, etc - as well as any metadata set created (sdPublisher)


> At the moment, there is no way to differentiate between these and datasets that have already been published elsewhere. Therefore, it wouldn't be appropriate to include this information in the templates.

I don't understand this statement. How can we not distinguish them?


> It would be helpful to discuss with @dblear and @pieterprovoost whether, and if so, how, this information should be incorporated.

Yes, it should be. The "how" we can figure out with an example or two.

**Comment by pbuttigieg** [comment]
> Marco Bolo should be added to all the templates as being the project under which the data were aggregated.
> @dblear can recommend the value to add (the URL that points to MBO, or whether we just use the string MARCO-BOLO), but meanwhile @laurianvm can you decide what the term should be?
> Something like partOf, usedIn

Examples from the WPs will help here. The type and subject of the record will likely determine the property.

Aggregations of datasets, for example, are themselves datasets with many parts. Each aggregation should have MBO's `@ResearchProject` metadata in a property like `creator`.

We should create an MBO JSON-LD record describing the project so we can use its PID (W3ID) in the `@id` keyword, rather than repeating the stanza for each asset.

**Comment by kmexter** [comment]
yes, we should have an MBO record, agreed. Referencing it will be necessary for datasets and software created under MBO as the "funder"

**Comment by pbuttigieg** [comment]
> yes, we should have an MBO record, agreed. Referencing it will be necessary for datasets and software created under MBO as the "funder" 

I'll create it (@roblinksdata this can be our first w3id) 

MBO is not the funder, that would be the Commission's Horizon instrument. The MBO record can be the value of properties like `creator', 'contributor', etc

**Comment by kmexter** [comment]
the commision's horizon instrument = MBO. I mean, MBO will need to be linked to its horizon grant number, and that is what is funding the work, so they are the same

**Comment by pbuttigieg** [comment]
> the commision's horizon instrument = MBO. I mean, MBO will need to be linked to its horizon grant number, and that is what is funding the work, so they are the same

They are linked, but not the same - the example will illustrate this. Also see: 

https://github.com/iodepo/odis-in/pull/12

```json

        "@type": "ResearchProject", 
         "name": "Global cooperation on FAIR data policy and practice", 
         "alternateName": "The WorldFAIR Project", 
         "url": "https://worldfair-project.eu/", 
         "identifier": "https://cordis.europa.eu/project/id/101058393" 
       }, 
       "recipient": { 
         "@type": "Organization", 
         "name": "The International Oceanographic Data and Information Exchange", 
         "url": "https://www.iode.org", 
         "identifier": "https://oceanexpert.org/institution/10171"

```

**Comment by laurianvm** [comment]
Hi @pbuttigieg, just checking if this is now resolved and whether we can close the issue. Thanks!

**Comment by pbuttigieg** [comment]
Thanks for the ping, we can resolve this via @roblinksdata 's csv2json code, if we auto-inject a generic credit to MBO (which we will have a dedicated JSON-LD file for) 

additional provenance can of course be added 

I'll create an issue on that repo and close this one

**Comment by pbuttigieg** [comment]
Subsumed by  https://github.com/marco-bolo/csv-to-json-ld/issues/4
